src/connectors/idagio.js

```diff
@@ -23,7 +23,11 @@ Connector.isPlaying = () => Util.getTextFromSelectors(pauseButtonSelector) === '
 Connector.isScrobblingAllowed = () => Util.getTextFromSelectors('.player-PlayerInfo__recordingInfo--15VMv') !== 'Sponsor message';
 
 function getCurrentTrack() {
-	return Util.getTextFromSelectors(trackSelector).split(' – ').slice(1).join(': ');
+	let track = Util.getTextFromSelectors(trackSelector).split(' – ').slice(1).join(': ').trim();
```

Comment:
It's not clear what (and why) transformations are performed. Could you add a comment with some examples?

Reply:
Sorry if I was not clear enough. I was talking about splitting and joining a track title, not about removing a dash. I know this line wasn't added in this PR, but I think it's better to have a comment describing why this transformation is added.

Reply:
My bad, I should've looked closer. Added now